### PR TITLE
Resource methods: add signatures, minor improvements

### PR DESCRIPTION
**Pull request by deining**
This PR improves the documentation of hugo pipes by adding method signatures for the available `resources.nn` functions. It also comes with minor textual  improvements.

### CODE REVIEW

File: content/en/hugo-pipes/postcss.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ menu:
     weight: 40
 toc: true
 weight: 40
+signature: ["resources.PostCSS RESOURCE [OPTIONS]", "postCSS RESOURCE [OPTIONS]"]
 ---
 
+### Usage
```
Comment (jmooring) [inline content/en/hugo-pipes/postcss.md:15]:
```suggestion
```

File: content/en/hugo-pipes/postcss.md [hunk 2]
```diff
@@ -9,8 +9,11 @@ menu:
...
+
```
Comment (jmooring) [inline content/en/hugo-pipes/postcss.md:16]:
```suggestion
```

File: content/en/hugo-pipes/transform-to-css.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Sass / SCSS
-description: Hugo Pipes allows the processing of Sass and SCSS files.
+title: ToCSS
+linkTitle: Transform Sass / SCSS
+description: Transformes an Sass or SCSS file into a CSS file.
```
Comment (jmooring) [inline content/en/hugo-pipes/transform-to-css.md:4]:
```suggestion
description: Transpile Sass to CSS.
```

File: content/en/hugo-pipes/transform-to-css.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
-title: Sass / SCSS
-description: Hugo Pipes allows the processing of Sass and SCSS files.
+title: ToCSS
+linkTitle: Transform Sass / SCSS
```
Comment (jmooring) [inline content/en/hugo-pipes/transform-to-css.md:3]:
```suggestion
linkTitle: Transpile Sass to CSS
```

**Review by jmooring — CHANGES_REQUESTED** [review]
Thanks for this. In addition to the minor changes I have requested, I have two other comments:

1) The first heading on some of the pages is `### Usage`. It may look better to have a smaller font size, but it we enable a multilevel TOC (H2, H3, etc.) the structure will be wrong.

2) I understand why you changed the page titles, but when I click on **Asset Bundling** in the sidebar, the page title is **Concat**. That makes sense, but was unexpected. So we have a mismatch between titles, link titles, and URLs.

There's some prep work underway for a new reference, where functions are name-spaced. Although not template functions, per se, perhaps we can group all of the resource.Foo methods in one place. It's on my list.

File: content/en/hugo-pipes/postcss.md (hunk 1 above)
Comment (deining) [reply content/en/hugo-pipes/postcss.md]: Removed the line as requested.

File: content/en/hugo-pipes/postcss.md (hunk 2 above)
Comment (deining) [reply content/en/hugo-pipes/postcss.md]: Removed the line as requested.

File: content/en/hugo-pipes/transform-to-css.md (hunk 3 above)
Comment (deining) [reply content/en/hugo-pipes/transform-to-css.md]: Changed as requested.

File: content/en/hugo-pipes/transform-to-css.md (hunk 4 above)
Comment (deining) [reply content/en/hugo-pipes/transform-to-css.md]: Changed as requested.

**Comment by deining** [comment]
> Thanks for this. In addition to the minor changes I have requested, I have two other comments:
> 
>     1. The first heading on some of the pages is `### Usage`. It may look better to have a smaller font size, but it we enable a multilevel TOC (H2, H3, etc.) the structure will be wrong.

I see. I chose `###` in all cases since the small font size looks better.

>     2. I understand why you changed the page titles, but when I click on **Asset Bundling** in the sidebar, the page title is **Concat**. That makes sense, but was unexpected. So we have a mismatch between titles, link titles, and URLs.

I changed the link title to `Concatenating assets` so that link title and title are more similar.
 
> There's some prep work underway for a new reference, where functions are name-spaced. Although not template functions, per se, perhaps we can group all of the resource.Foo methods in one place. It's on my list.

Thanks for your continued documentation efforts!

**Comment by jmooring** [comment]
> The first heading on some of the pages is `### Usage`. It may look better to have a smaller font size, but it we enable a multilevel TOC (H2, H3, etc.) the structure will be wrong.

Start with H2.

**Comment by deining** [comment]
> > The first heading on some of the pages is `### Usage`. It may look better to have a smaller font size, but it we enable a multilevel TOC (H2, H3, etc.) the structure will be wrong.
> 
> Start with H2.

Done.

**Pull request merged**